test/api_adapter.test.js

```diff
@@ -1,12 +1,12 @@
-const test = require("ava");
-const sinon = require("sinon");
-const apiAdapter = require("../api_adapter");
-const superagent = require("superagent");
+const test = require('ava');
+const sinon = require('sinon');
+const { getTextFrom } = require('../api_adapter');
```

Comment:
Any reason why you prefer this syntax over something like, `const getTextFrom = require('../api_adapter').getTextFrom` . Is it purely for brevity or is there some other advantage?

Reply:
Good practice. This way multiple members could be destructured out in one place.